src/Console/Output/ErrorOutput.php

```diff
@@ -55,49 +56,62 @@ public function listErrors($process, array $errors)
+            if (!$showDetails || null === $e) {
+                continue;
+            }
 
-                    $length = max(strlen($class), strlen($message));
-                    $lines = [
-                        '',
-                        $class,
-                        $message,
-                        '',
-                    ];
+            $class = sprintf('[%s]', get_class($e));
+            $message = $e->getMessage();
+            $code = $e->getCode();
+            if (0 !== $code) {
+                $message .= " (${code})";
+            }
 
-                    $this->output->writeln('');
+            $length = max(strlen($class), strlen($message));
+            $lines = [
+                '',
+                $class,
+                $message,
+                '',
+            ];
 
-                    foreach ($lines as $line) {
-                        if (strlen($line) < $length) {
-                            $line .= str_repeat(' ', $length - strlen($line));
-                        }
+            $this->output->writeln('');
 
-                        $this->output->writeln(sprintf('      <error>  %s  </error>', $this->prepareOutput($line)));
-                    }
+            foreach ($lines as $line) {
+                if (strlen($line) < $length) {
+                    $line .= str_repeat(' ', $length - strlen($line));
+                }
 
-                    if ($showTrace && !$e instanceof LintingException) { // stack trace of lint exception is of no interest
-                        $this->output->writeln('');
-                        $stackTrace = $e->getTrace();
-                        foreach ($stackTrace as $trace) {
-                            if (isset($trace['class'], $trace['function']) && \Symfony\Component\Console\Command\Command::class === $trace['class'] && 'run' === $trace['function']) {
-                                $this->output->writeln('      [ ... ]');
+                $this->output->writeln(sprintf('      <error>  %s  </error>', $this->prepareOutput($line)));
+            }
 
-                                break;
-                            }
+            if ($showTrace && !$e instanceof LintingException) { // stack trace of lint exception is of no interest
+                $this->output->writeln('');
+                $stackTrace = $e->getTrace();
+                foreach ($stackTrace as $trace) {
+                    if (isset($trace['class'], $trace['function']) && \Symfony\Component\Console\Command\Command::class === $trace['class'] && 'run' === $trace['function']) {
+                        $this->output->writeln('      [ ... ]');
 
-                            $this->outputTrace($trace);
-                        }
+                        break;
                     }
+
+                    $this->outputTrace($trace);
                 }
             }
+
+            if (Error::TYPE_LINT === $error->getType() && $e instanceof LintingException) {
```

Comment:
In this file, this if is the real change.

Above there is only an indentation change due to this minor enhancement:
```diff
-if ($showDetails) {
-    $e = $error->getSource();
-    if (null !== $e) {
+$e = $error->getSource();
+if (!$showDetails || null === $e) {
+    continue;
```